src/main/java/net/dries007/tfc/objects/blocks/stone/BlockButtonStoneTFC.java

```diff
@@ -28,6 +29,12 @@ public BlockButtonStoneTFC(Rock rock)
     {
         this.rock = rock;
         if (MAP.put(rock, this) != null) throw new IllegalStateException("There can only be one.");
+
+        OreDictionaryHelper.register(this, "button");
+        OreDictionaryHelper.register(this, "button", "stone");
+        OreDictionaryHelper.register(this, "button", "stone", rock.getRegistryName().getPath());
```

Comment:
I'm with clay here - I think specifying the rock category / rock are a bit excessive. Especially for something like buttons. But I'm open to why you think its necessary.